.gitignore

```diff
@@ -21,6 +21,7 @@ dub.selections.json
 /compiler/src/vcbuild/Win32
 /compiler/src/vcbuild/x64
 /compiler/src/vcbuild/.vs
+/compiler/src/bug2
```

Comment:
What's this for?

Reply:
It's a junk directory on my system that git suddenly started complaining about.

Reply:
`git add -p` is your friend

Reply:
Otherwise, you can also add it to your `.git/info/exclude` this is a gitignore for you only (not shared)

Reply:
@WalterBright please take care of this so we can merge this.